control-plane/plugin/src/bin/rest-plugin/main.rs

```diff
@@ -72,11 +72,22 @@ async fn execute(cli_args: CliArgs) {
                 )
                 .await
             }
+            GetResources::CordonLabels { id } => node::Node::get_labels(id).await,
```

Comment:
Do we need a resource here? Otherwise if in future we have pool cordoning feature, then this probably won't support that. Rather should we just associate the command to a resource itself?